backend/README.md

```diff
@@ -10,15 +10,25 @@ Create a `.env` file in this directory.
 
 ```
 JWT_SECRET=<Your Secret>
+process.env.NEO4J_USERNAME=<Neo4J Username>
+process.env.NEO4J_PASSWORD=<Neo4J Password>
 ```
 
 ### Run the server
 
+run Neo4J with Docker
+
+```
+$ docker run  --publish=7474:7474 --publish=7687:7687 --env 'NEO4JLABS_PLUGINS=["apoc"]' --env=NEO4J_AUTH=none neo4j:latest
+```
+
+run the backend
+
 ```
 npm run serve
 ```
 
-### Lints and fixes files
+### Lints files
 
 ```
 npm run lint
```

Comment:
very nice! you have decided for the scenario of neo4j, we also have it. But maybe you are missing the explanation why you have chosen this. Also an objective ⭐ For explaining WHY you have chosen the scenario in the README.md.❌

Reply:
You are right, we totally forgot to explain it. Thanks for pointing it out to us 👍